fe/lib/components/QuestionnaireBuilder/QuestionnaireBuilder/FormBuilder/components/NewPrivacyConsentForm/NewPrivacyConsentForm.tsx

```diff
@@ -70,24 +71,32 @@ export default ({
             </Heading>
           </Box>
           <Controller
-            as={TextField}
+            render={(formProps) => (
```

Comment:
Curious is the form props typed when you pull them off in this render function?

Reply:
Yes, `react-hook-form` is providing some loose types here:

https://github.com/react-hook-form/react-hook-form/blob/6a120e4165b7fdf724004e327b2c4dc87be31dd5/src/types/props.ts#L44-L48